bin/docker_run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# Setup yarn and build qmk_configurator inside the docker container.
+
+set -e
+#set -x
+
+# Set the API URL
+sed "s;%API_URL%;${QMK_API_URL:-https://api.qmk.fm};g" /etc/nginx/nginx.conf.in > /etc/nginx/nginx.conf
```

Comment:
Other than using `sed` on the config file why not using built in nginx way to handle env var?
https://web.archive.org/web/20170712003702/https://docs.apitools.com/blog/2014/07/02/using-environment-variables-in-nginx-conf.html#with-perl